osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -207,6 +208,22 @@ private void load(LocalisationEngine localisation)
                         }
                     },
                     new FillFlowContainer
+                    {
+                        Name = "Topright-aligned metadata",
+                        Anchor = Anchor.TopRight,
+                        Origin = Anchor.TopRight,
+                        Direction = FillDirection.Vertical,
+                        Margin = new MarginPadding { Top = 14, Left = 10, Right = 18, Bottom = 20 },
+                        AutoSizeAxes = Axes.Both,
+                        Children = new Drawable[]
+                        {
+                            StatusPill = new BeatmapSetOnlineStatusPill(11, new MarginPadding { Horizontal = 8, Vertical = 2 })
```

Comment:
Move margins into class. They are the same in every usage.

Reply:
Only 2 of 4 usages happen to have the same values.